src/styles/shared/_buttons.scss

```diff
@@ -247,6 +248,7 @@
       background: transparent;
       box-shadow: 0 0 0 border-width('base') var(--p-critical-border);
       color: var(--p-critical-link);
+      @include recolor-icon(var(--p-critical-link));
```

Comment:
Again, parity between destructive button icon and text colors. cc @jessebc and @mirualves

Reply:
Yeah, the after makes sense to me. The contrast looks much better in both light and dark mode.

Reply:
👀 Just a random question, but why are the outline and the text for the dark mode button two separate colors?

Reply:
The short answer is for contrast ratios. But it is possible these colors will still be tweaked—it is more important the code is using the proper color role variable.